nixos/modules/services/hardware/amdgpu.nix

```diff
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.amdgpu;
+in {
+  options.hardware.amdgpu = {
+    legacySupport.enable = lib.mkEnableOption ''
+      support for old AMD graphics cards, this uses `amdgpu`
+      instead of `radeonsi` driver for Southern Islands (Radeon HD 7000)
+      series and Sea Islands (Radeon HD 8000) series cards.
+      Note: this removes support for analog video output graphics cards,
+      which is only available in `radeonsi` driver
```

Comment:
This is still wrong. It does not remove support for "analog output video cards", it removes support for _analog outputs_, which can coexist with digital outputs on the same hardware (and in fact basically always do - I can't think of a single analog-only SI/CIK design in the wild).

Reply:
Thanks for the feedback, I'm out of city right now, will get to these as soon as I can! Or you can push as a commiter :)